plugins/destiny.py

```diff
@@ -343,6 +346,20 @@ def triumph(text, nick, bot):
 
 @hook.command('xur')
 def xur(text, bot):
+    now = datetime.datetime.now(DAILY_RESET_TIMEZONE)
```

Comment:
this line would accomplish what youwant, and remove the need for the global variable.
```
now = datetime.datetime.utcnow() - datetime.timedelta(hours=-9)
```